syncer/syncer_test.go

```diff
@@ -968,12 +969,28 @@ func (s *testSyncerSuite) TestcheckpointID(c *C) {
 }
 
 func (s *testSyncerSuite) TestCasuality(c *C) {
-	var wg sync.WaitGroup
+	p := parser.New()
+	se := mock.NewContext()
+	schema := "create table tb(a int primary key, b int unique);"
+	ti, err := createTableInfo(p, se, int64(0), schema)
+	c.Assert(err, IsNil)
+	insertValues := [][]interface{}{
+		{1, 2},
+		{3, 4},
+		{4, 1},
+		{1, 4}, // this insert conflict with the first one
```

Comment:
Maybe delete the old values before this insert makes more sense.

Reply:
This unit test doesn't execute the sql, it just uses the value of the sql to gen key
so there is no way to execute the delete statement for now🤣